playbooks/common/openshift-master/config.yml

```diff
@@ -372,6 +372,12 @@
   - role: flannel_register
     when: openshift.common.use_flannel | bool
 
+- name: Configure Nuage 
+  hosts: oo_first_master
+  roles:
+  - role: nuage_master 
+    when: openshift.common.use_nuage | bool
```

Comment:
Is there a reason this would only be installed on a single master? Ideally, the master component would be able to be installed on all masters to avoid a single point of failure.